pom.xml

```diff
@@ -147,28 +153,6 @@
 			</plugins>
 		</pluginManagement>
 		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-checkstyle-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>checkstyle-validation</id>
-						<phase>validate</phase>
-						<goals>
-							<goal>check</goal>
-						</goals>
-						<configuration>
-							<!-- Checkstyle rules inherited from spring-cloud-build -->
-							<suppressionsLocation>src/checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
```

Comment:
I'm pretty sure we are still in the situation when we would like to suppress some rules.
So, what is new way to do that?

Reply:
we automatically pick `root/src/checkstyle/checkstyle-suppressions.xml` file, so you don't need to explicitly provide it. All is described here - https://github.com/spring-cloud/spring-cloud-build#checkstyle